how-to-make-new-posts.md

```diff
@@ -0,0 +1,31 @@
+Posts Creation
+
+If you want to create a post you should create a file in _posts, the file name structure need to be YYYY-MM-DD-name-of-the-post.md)
+
+![Screen](/images/postsinvestigation/1.png)
+
+This is what  we need to add to our new post:
+    1.- published: true (or “false” if you don’t want this post to be published)
```

Comment:
This list cannot be seen properly, you can check it here:
https://github.com/juanjoseortega-aircury/Sembahome/blob/SH-13-MD-file-posts-investigation/how-to-make-new-posts.md